### PR TITLE
enable galaxy-targeting

### PR DESCRIPTION
**Pull request by mcoughlin**
This PR enables galaxy-targeted follow-up for MMA scheduling.

### CODE REVIEW

File: skyportal/facility_apis/observation_plan.py [hunk 1]
```diff
@@ -144,15 +150,52 @@ def generate_plan(observation_plan_id, request_id):
...
+        if params["tilesType"] == "galaxy":
+            query = Galaxy.query_records_accessible_by(user, mode="read")
+            query = query.filter(Galaxy.catalog_name == params["galaxy_catalog"])
+            galaxies = query.all()
+            catalog_struct = {}
+            catalog_struct["ra"] = np.array([g.ra for g in galaxies])
+            catalog_struct["dec"] = np.array([g.dec for g in galaxies])
+            catalog_struct["S"] = np.array([1.0 for g in galaxies])
+            catalog_struct["Sloc"] = np.array([1.0 for g in galaxies])
+            catalog_struct["Smass"] = np.array([1.0 for g in galaxies])
```
Comment (guynir42) [inline skyportal/facility_apis/observation_plan.py:162]: Did you intend to leave 1.0 as the value for all these?

**Review by guynir42 — APPROVED** [review]
I don't know how this happened but it all looks good.

File: skyportal/facility_apis/observation_plan.py (hunk 1 above)
Comment (mcoughlin) [reply skyportal/facility_apis/observation_plan.py:162]: Yeah because will need to think about how to properly treat galaxy metric weighting.

**Comment by mcoughlin** [comment]
Thanks @guynir42.

**Pull request merged**